content/source/docs/enterprise/sentinel/manage-policies.html.md

```diff
@@ -1,36 +1,64 @@
 
-### Organizing Policies With Policy Sets
+During a Terraform run, Terraform Enterprise (TFE) checks all of the Sentinel policies that apply to the run's workspace. This includes the members of any global policy sets, and of any policy sets that are explicitly applied to that workspace.
 
-~> **Important:** In order for Sentinel policies to be checked during runs, they must be added to at least one policy set that has been attached to a workspace. Policies that are not a member of a policy set, or are a member of a policy set that has not been attached to any workspaces, will not be checked during any runs in any workspace.
+~> **Important:** In order for a policy to take effect, it must be in one or more policy sets. Individual policies cannot be assigned to workspaces, and policies that are not in any policy sets will never be checked.
 
-To manage your organization's policy sets, navigate to the organization's settings page under "Policy Sets". From the "Policy Sets" section you can view, list, create, edit, and delete policy sets. Editing a policy set allows you to add or remove policies from the set, or attach or detach the policy set from workspaces.
+Policies and policy sets are managed at an organization level, and only [organization owners](../users-teams-organizations/teams.html#the-owners-team) can create, edit or delete them.
 
-To create a new policy set, navigate to "Create Policy Set".
+-> **Note:** The UI controls for managing Sentinel policies are designed for demos and other simple use cases; for complex usage with multiple policies, we recommend [storing Sentinel code in version control](./integrate-vcs.html) and using your CI system to test and upload them via TFE's API. In the future, Terraform Enterprise will integrate directly with VCS providers for the Sentinel workflow.
 
-- **Policy Set Name**: The name of your policy set is used in the UI and must be unique to your organization. Accepts alphanumeric characters, as well as `-` and `_`.
+## Managing Policies
+
+[organization settings]: ../users-teams-organizations/organizations.html#organization-settings
+
+To manage an organization's Sentinel policies, go to the [organization settings][] and navigate to the "Policies" page.
+
+To create a new policy, click the "Create a new policy" button; to edit an existing policy, click its entry in the list. Be sure to click the "Create policy" or "Update policy" button when finished.
+
+When creating or editing a policy, the following fields are available:
+
+- **Policy Name:** The name of your policy, which is used in the UI and Sentinel output. Accepts letters, numbers, `-`, and `_`. (Names cannot be modified after creation, and this field is disabled when editing an existing policy.)
+- **Description:** A brief note about the policy's purpose, displayed in the policy list.
+- **Enforcement Mode:** The level to which the policy is enforced when performing a run. The following modes are available:
```

Comment:
```suggestion
- **Enforcement Mode:** The level to which the policy is enforced when performing a run. The following levels are available:
```

Reply:
Nah, I'm leaving this one so it matches the UI/API label ("enforcement modes").

Reply:
Can we ditch the use of "level" entirely, then, and just say something like "The way that"?

I don't feel super strongly, but using both level and mode seems a bit confusing.

Reply:
Sure, that seems like a good idea. 

```suggestion
- **Enforcement Mode:** How the policy is enforced when performing a run. The following modes are available:
```